cli/azd/cmd/env.go

```diff
@@ -209,7 +209,7 @@ func (en *envNewAction) Run(ctx context.Context, cmd *cobra.Command, args []stri
 		subscription:    en.subscription,
 		location:        en.location,
 	}
-	if _, _, err := createAndInitEnvironment(ctx, &envSpec, azdCtx, console); err != nil {
+	if _, _, err := createAndInitEnvironment(ctx, &envSpec, azdCtx, console, cmd.Parent().OutOrStdout()); err != nil {
```

Comment:
This looks like some autocomplete badness, should likely be just cmd.OutOrStdOut()